### ISSUE TITLE
please support region eu-north

### ISSUE THREAD

**Issue by jave** [openshift/installer#1814]
Summary:

We want to install a openshift 4.1 cluster in region eu-north (stockholm). This region is currently not supported. We were able to install a test cluster in AWS region eu-central-1, Frankfurt, just fine.

We would like our production cluster to be in eu-north-1.

# Version

4.1

```console
$ openshift-install version
<your output here>
```

# Platform (aws|libvirt|openstack):

AWS, region eu-north

# What happened?

region is not supported

Enter text here.
See [the troubleshooting documentation](https://github.com/openshift/installer/blob/master/docs/user/troubleshooting.md) for ideas about what information to collect.
For example, if the installer [fails to create resources](https://github.com/openshift/installer/blob/master/docs/user/troubleshooting.md#installer-fails-to-create-resources), attach the relevant portions of your `.openshift_install.log`.

# What you expected to happen?

we expected it to be possible to install an openshift cluster on aws in region eu-north (stockholm)

# How to reproduce it (as minimally and precisely as possible)?

```console
$ your-commands-here
```

# Anything else we need to know?

Enter text here.

# References

- enter text here.

**Comment by attfr** [comment]
I hope they will support the eu-north region as soon as possible

**Comment by cgwalters** [comment]
https://url.corp.redhat.com/70092ad is a (RHT-internal) link to a change to the buildsystem to add this region.

**Comment by attfr** [comment]
Hi cgwalters 
When i'm testing  openshift-install-linux-4.2.0-0.nightly-2019-09-02-172410.tar.gz ther is not eu-north region in the installation. Are you sure that the eu-north comes as an alternative in openshift 4.2?

Best regads 
Fredrik

**Comment by cgwalters** [comment]
Hm, it should be supported now, we definitely have bootimages, etc.:
https://github.com/openshift/installer/blob/3ab66dec4cbcdc7339e05f9e5a6e27dac3229e47/data/data/rhcos.json#L24

**Comment by attfr** [comment]
Then I have to wait and see if it shows up in the actual release for 4.2. And thank you for fast answer. 

//Fredrik

**Comment by wking** [comment]
Looks like it was added to `rhcos.json` in #1941.  But we still need a manual bump or #1582 to get it through validation and into wizard selection.  Since #1582 has been open for a while, I'll file the smaller bump as well.

**Comment by wking** [comment]
Oops, I meant #1528.

**Comment by wking** [comment]
> ... and then will require backporting to release-4.1.

Previous discussion has suggested that we will not backport support to additional regions to 4.1.z releases.  For example, see [here][1].

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=1734136#c4

**Comment by attfr** [comment]
So if i wait for 4.2 then that version will support eu-north in aws?

**Comment by wking** [comment]
If something like #1528 or #2308 lands before 4.2, yes.

**Comment by attfr** [comment]
Thank you very much for the information. I keep my thumbs that it will be included in the 4.2 release

Best regads
Fredrik
